### PR TITLE
SDA-5813 - Remove DisplayName/display_name from cluster

### PR DESCRIPTION
**Pull request by hunterkepley**
https://issues.redhat.com/browse/SDA-5813

### CODE REVIEW

**Comment by pvasant** [comment]
@zgalor PTAL

**Comment by vkareh** [comment]
We still have `name` which is what customers name the cluster to begin with. Do we want to use that instead? In `rosa`, for example, we allow cluster selection by name: `$ rosa describe cluster -c mycluster-foo`

So maybe we want to use `name` instead of removing `display_name`

**Comment by hunterkepley** [comment]
> We still have `name` which is what customers name the cluster to begin with. Do we want to use that instead? In `rosa`, for example, we allow cluster selection by name: `$ rosa describe cluster -c mycluster-foo`
> 
> So maybe we want to use `name` instead of removing `display_name`

Added `name = '%s'` to replace `display_name = '%s`.

Thanks for letting me know this!

**Comment by hunterkepley** [comment]
Tested and `DisplayName` no longer shows when describing a cluster through ocm-cli

File: pkg/cluster/cluster.go [hunk 1]
```diff
@@ -147,7 +147,7 @@ func GetCluster(connection *sdk.Connection, key string) (cluster *cmv1.Cluster,
 
 	// Try to find a matching subscription:
 	subsSearch := fmt.Sprintf(
-		"(display_name = '%s' or cluster_id = '%s' or external_cluster_id = '%s') and "+
+		"(name = '%s' or cluster_id = '%s' or external_cluster_id = '%s') and "+
```
Comment (zgalor) [inline pkg/cluster/cluster.go:150]: @hunterkepley this is an AMS field. Shouldn't change.
fixed in https://github.com/openshift-online/ocm-cli/pull/413

Comment (zgalor) [reply pkg/cluster/cluster.go:150]: cc @vkareh

**Pull request merged**